stablehlo/dialect/StablehloOps.td

```diff
@@ -1604,11 +1604,15 @@ def StableHLO_SliceOp: StableHLO_Op<
 
     Example:
     ```mlir
+    %result = stablehlo.slice %operand [1:3, 4:8:1]
+       : (tensor<3x8xi64>) -> tensor<2x2xi64>
+
+    // Same in generic form
     %result = "stablehlo.slice" (%operand) {
-      start_indices = dense<[1, 2]> : tensor<2xi64>,
-      limit_indices = dense<[3, 4]> : tensor<2xi64>,
-      strides = dense<1> : tensor<2xi64>
-    } : (tensor<3x4xi64>) -> tensor<2x2xi64>
+      start_indices = dense<[1, 4]> : tensor<2xi64>,
+      limit_indices = dense<[3, 8]> : tensor<2xi64>,
+      strides = dense<[1, 2]> : tensor<2xi64>
+    } : (tensor<3x8xi64>) -> tensor<2x2xi64>
```

Comment:
I don't really care about the generic syntax: to me it was a shortcut to explain the in-memory representation and avoid having to write a paragraph about it :)

Reply:
Sounds good - I'm okay with breaking the convention here, since this is done for documentation reasons. Later on, we'll find a dedicated place to document the pretty syntax.